addons/web/static/src/js/views/list/list_controller.js

```diff
@@ -599,7 +616,7 @@ var ListController = BasicController.extend({
      * @returns {Promise}
      */
     _toggleArchiveState: async function (archive) {
-        const resIds = await this.getSelectedIdsWithDomain()
+        const resIds = await this.getSelectedRecordsWithDomain().then(res => res.map(record => record.id));
```

Comment:
why not keep using the former function in this case?

Reply:
The test here fails because of the change in the rpc call https://github.com/odoo/odoo/blob/794d8eec260296a2af33bcb02eb2d2cc1d14406f/addons/web/static/tests/views/list_tests.js#L665

Reply:
Nop, I was the one missing something. Thanks!